doc/changelogs/CHANGELOG_V12.md

```diff
@@ -53,6 +54,21 @@
   * [io.js](CHANGELOG_IOJS.md)
   * [Archive](CHANGELOG_ARCHIVE.md)
 
+<a id="12.16.1"></a>
+## 2020-02-13, Version 12.16.1 'Erbium' (LTS), @MylesBorins
+
+### Notable changes
+
+Node.js 12.16.0 included large update to the ESM implementation in the 12.x release stream.
+One of the new features, Self Referential Modules, accidentally was released not behind the
```

Comment:
```suggestion
One of the new features, Self Referential Modules, was accidentally released without requiring the
```

Reply:
thanks rich. I'll make sure this is also reflected in the commit message